actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Add the following permissions policy directives: `hid`, `idle-detection`, `screen-wake-lock`, `serial`, `sync-xhr`, `web-share`.
```

Comment:
Thanks! Could you wrap them at around 80 characters?
https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#updating-the-changelog
```suggestion
*   Add the following permissions policy directives: `hid`, `idle-detection`, `screen-wake-lock`,
    `serial`, `sync-xhr`, `web-share`.
```